splice/web/api/campaign.py

```diff
@@ -45,12 +44,27 @@
     'account_id', type=int, required=True, help='Account ID', location='json')
 
 
+def Bool(s):
```

Comment:
Flask-restful has a few built-in functions available, which take care of error handling as well.
```python
from flask_restful import inputs

self.reqparse_get.add_argument('past', type=inputs.boolean, ...
self.reqparse_get.add_argument('today', type=inputs.date, ...
```